playbooks/roles/edxapp/templates/edx/app/edxapp/worker.sh.j2

```diff
@@ -7,6 +7,18 @@ source {{ edxapp_app_dir }}/edxapp_env
 {% if COMMON_ENABLE_NEWRELIC_APP %}
 {% set executable = edxapp_venv_bin + '/newrelic-admin run-program ' + edxapp_venv_bin + '/celery' %}
 
+{% if COMMON_ENABLE_DATADOG and COMMON_ENABLE_DATADOG_APP %}
+{% set executable = edxapp_venv_bin + '/ddtrace-run ' + executable %}
+export DD_TAGS=$(dd_tags)
```

Comment:
I'm not entirely following how the `--dd-tags` value ends up in here.

Reply:
Its being passed as an arg when the script is run in workers.conf.j2, in the same way that hostname is several lines above. Is that not the right way to do it?

Reply:
`$(hostname)` in a shell script means it runs `hostname` as a command and inserts the stdout as the string. I believe all of the args being passed in the conf file end up being interpreted by Celery -- see the `exec {{ executable }} $@` at the end of this script.

Reply:
Ok I figured out a way to remove that arg from the args list once I use it.